src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -0,0 +1,368 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlTypes;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Numerics;
+using System.Text;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public partial class BinaryFormatterTests
+    {
+        /// <summary>
+        /// First: Object
+        /// Second: Core Base64 serialized string
+        /// Third: Netfx Base64 serialized string 
```

Comment:
I thought a lot about what the best data structure could be for this case. Putting the base64 strings into a different container for every configuration brings the hassle of references to the objects. The base64 hash of e.g. byte.MaxValue must reference the value byte.MaxValue. Therefore I decided to put them together in one place.
If we e.g. need a dedicated hash for net471 we could just add another object to the array.